x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/actions/duplicate_exceptions.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Should we move this files to `detection_engine/rule_exceptions`? It's logic specific to exceptions and having it here will add it to Detection Rules CODEOWNERS files

Reply:
not sure here, it's only used in duplicateRules which is located in this folder and belongs to actions.